mne/preprocessing/maxwell.py

```diff
@@ -505,3 +553,113 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _read_fine_cal(info, fine_cal_fname):
+    """Read sensor locations and calib. coeffs from fine calibration file."""
+
+    logger.info('Fine calibration file used in Maxwell filter: ' +
+                fine_cal_fname)
```

Comment:
This `logger.iinfo` should probably actually be in `maxwell_filter`, not in the fine cal reading. That way this function could be used by something (anything) else that needs to read that type of file. Think orthogonality and hopefully it makes sense.